packages/crud/theme/material/vaadin-crud-styles.js

```diff
@@ -116,8 +118,15 @@ registerStyles(
 registerStyles(
   'vaadin-crud-dialog-overlay',
   [
+    overlay,
+    dialogOverlay,
     editorStyles,
     css`
+      [part='overlay'] {
+        max-width: 54em;
+        min-width: 20em;
+      }
```

Comment:
Previously, `vaadin-crud` styles from `src` folder were applied after `material` style module because of `CrudDialogOverlay` extending `DialogOverlay`. This is no longer the case, so the order has changed. As a result, the following CSS rules from the `vaadin-dialog` Material folder started to apply:

https://github.com/vaadin/web-components/blob/c2421a9017d3f90b493c28cefd18b6d56c500222/packages/dialog/theme/material/vaadin-dialog-styles.js#L10-L11